src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Extends Command class
+ * When executed, calls Ui method to print output
+ */
+public class AddCommand extends Command {
+    private Task task;
+
+    /**
+     * Constructor
+     * Initialise task attribute to the input task to be added
+     * @param task Task to be added
+     */
```

Comment:
```suggestion
    /**
     * Constructor
     * Initialise task attribute to the input task to be added
     *
     * @param task Task to be added
     */
```

Good job on all the Javadoc! However, do take note that it is standard convention to leave an empty line between the description field and parameter/return/throw fields.